content/download/docker.en.md

```diff
@@ -19,51 +19,55 @@ For a version matrix (GRASS GIS, PROJ, GDAL, PDAL), see [here](https://github.co
+*  [<i class="fa fa-download"></i> GRASS GIS 8.0.x, Ubuntu 18.04 based](https://hub.docker.com/r/mundialis/grass-py3-pdal) (<small>release branch, grass78, 1.3 GB, with Python 3 and PDAL</small>)
    <pre><code class="dockerfile">docker pull mundialis/grass-py3-pdal:stable-ubuntu</code></pre>
 
 
 <hr>
 
 
-### <a name="GRASS-GIS-old"></a> GRASS GIS 7.6.1 (legacy)
+### <a name="GRASS-GIS-old"></a> GRASS GIS 7.8.6 (legacy)
 
-<div class="alert rounded-0 alert-warning">
-<i class="fa fa-info-circle"></i> The following Docker images are generated from the <b>GRASS GIS releasebranch_7_6</b> (<a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures76">GRASS GIS 7.6.1 new features</a>)
-</div>
+<div class="alert rounded-0 alert-success">
+<i class="fa fa-info-circle"></i> The following Docker images are generated from the <b>GRASS GIS releasebranch_7_8</b> (<a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78">GRASS GIS 7.8 new features</a>)</div>
 
-*  [<i class="fa fa-download"></i> GRASS GIS 7.6 stable, Ubuntu 18.04 based](https://hub.docker.com/r/mundialis/grass-gis-stable) (<small style="display:inline-block;margin-bottom:1em;">release branch, grass76, 730 MB, with Python 2</small>)
-   <pre><code class=dockerfile">docker pull mundialis/grass-gis-stable</code></pre>
+*  [<i class="fa fa-download"></i> GRASS GIS 7.8.x, Alpine based](https://hub.docker.com/r/mundialis/grass-py3-pdal) (<small>release branch, grass78, 320 MB, with Python 3 and PDAL</small>)
+    <pre><code class=dockerfile">docker pull mundialis/grass-py3-pdal:stable-alpine</code></pre>
 
+*  [<i class="fa fa-download"></i> GRASS GIS 7.8.x, Debian 10 based](https://hub.docker.com/r/mundialis/grass-py3-pdal) (<small>release branch, grass78, 1.2 GB, with Python 3 and PDAL</small>)
+    <pre><code class="dockerfile">docker pull mundialis/grass-py3-pdal:stable-debian</code></pre>
+
+*  [<i class="fa fa-download"></i> GRASS GIS 7.8.x, Ubuntu 18.04 based](https://hub.docker.com/r/mundialis/grass-py3-pdal) (<small>release branch, grass78, 1.3 GB, with Python 3 and PDAL</small>)
+   <pre><code class="dockerfile">docker pull mundialis/grass-py3-pdal:stable-ubuntu</code></pre>
 
 <hr>
 
 
-### <a name="GRASS-GIS-devel"></a> GRASS GIS 8.0 (preview)
+### <a name="GRASS-GIS-devel"></a> GRASS GIS 8.2.0 (preview)
 
 <div class="alert rounded-0 alert-info">
-<i class="fa fa-info-circle"></i> The following Docker images are generated from the <b>GRASS GIS master branch</b>
+<i class="fa fa-info-circle"></i> The following Docker images are generated from the <b>GRASS GIS main branch</b>
 </div>
 
-*  [<i class="fa fa-download"></i> GRASS GIS 8.0 development, Ubuntu 18.04 based](https://hub.docker.com/r/neteler/grassgis7/) (<small>git main, grass80, 520 MB, with Python 3</small>)
+*  [<i class="fa fa-download"></i> GRASS GIS 8.2 development, Ubuntu 18.04 based](https://hub.docker.com/r/neteler/grassgis7/) (<small>git main, grass80, 520 MB, with Python 3</small>)
```

Comment:
This is outdated since dockerhub changed their policy to paid accounts.
Hence the entry may be removed.